gadgets/trace_open.bpf.c

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2019 Facebook
+// Copyright (c) 2020 Netflix
+#include <vmlinux/vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include "trace_open.h"
+
+#define TASK_RUNNING	0
+
+const volatile __u64 min_us = 0;
+const volatile pid_t targ_pid = 0;
+const volatile pid_t targ_tgid = 0;
+const volatile uid_t targ_uid = INVALID_UID;
+const volatile bool targ_failed = false;
```

Comment:
Those are not setup (RewriteConstants) in userspace. Do you intend to keep them and to enable gadget parameters in a future PR?

Reply:
Probably yes. I don't want to remove them from the eBPF code to avoid having to reintroduce them again.